static/src/js/components/ManageCitationAssociations/ManageCitationAssociations.jsx

```diff
@@ -0,0 +1,316 @@
+import { useMutation, useSuspenseQuery } from '@apollo/client'
+import React, { useCallback, useState } from 'react'
+import { useParams } from 'react-router'
+import { Alert } from 'react-bootstrap'
+
+import pluralize from 'pluralize'
+import Col from 'react-bootstrap/Col'
+import Row from 'react-bootstrap/Row'
+
+import Button from '@/js/components/Button/Button'
+import CustomModal from '@/js/components/CustomModal/CustomModal'
+import EllipsisText from '@/js/components/EllipsisText/EllipsisText'
+import Table from '@/js/components/Table/Table'
+import ControlledPaginatedContent from '@/js/components/ControlledPaginatedContent/ControlledPaginatedContent'
+
+import useAccessibleEvent from '@/js/hooks/useAccessibleEvent'
+import useNotificationsContext from '@/js/hooks/useNotificationsContext'
+
+import { DELETE_ASSOCIATION } from '@/js/operations/mutations/deleteAssociation'
+import { GET_CITATION_ASSOCIATIONS } from '@/js/operations/queries/getCitationAssociations'
+
+import errorLogger from '@/js/utils/errorLogger'
+
+/**
+ * Renders a ManageServiceAssociations component
+ *
+ * @component
+ * @example <caption>Render a ManageServiceAssociations</caption>
+ * return (
+ *   <ManageServiceAssociations />
+ * )
+ */
+const ManageCitationAssociations = () => {
```

Comment:
We should refactor this on the next concept that we end up adding to ensure that we don't have multiple code